contracts/shared/helpers/Fixture.ts

```diff
@@ -75,115 +40,89 @@ export default class Fixture {
     secretNumbers?: number[]
   ) {
     let chainId = (await ethers.provider.getNetwork()).chainId;
-    let provider = ethers.provider;
 
     let allSigners = await ethers.getSigners();
     let signers = (allSigners).slice(0, Fixture.ECDSA_ACCOUNTS_LENGTH);
     let addresses = await Promise.all(signers.map(acc => acc.getAddress())) as string[];
 
-    let blsSignerFactory = await BlsSignerFactory.new();
-    let blsSigners = new Array(blsWalletCount);
-    for (let i=0; i<blsSigners.length; i++) {
-      let secretNumber = Math.abs(Math.random() * 0xffffffff << 0);
-      if (secretNumbers) {
-        secretNumber = secretNumbers[i]; // error here if not enough numbers provided
-      }
-      blsSigners[i] = blsSignerFactory.getSigner(DOMAIN, "0x"+secretNumber.toString(16));
-    }
-
     let create2Fixture = Create2Fixture.create();
 
     // deploy wallet implementation contract
-    let blsWalletImpl = await create2Fixture.create2Contract("BLSWallet") as BLSWallet;
+    let blsWalletImpl = await create2Fixture.create2Contract("BLSWallet");
     try {
       await (await blsWalletImpl.initialize(
         ethers.constants.AddressZero
       )).wait();
     } catch (e) {}
 
     // deploy Verification Gateway
-    let verificationGateway = await create2Fixture.create2Contract("VerificationGateway") as VerificationGateway;
```

Comment:
VerificationGateway was the typechain type, and naming with lowercase contract name the convention for Contract instances.

Reply:
As discussed "VerificationGateway.ts" will be replaced with typechain type.